portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,58 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
+  * This is the main javascript file for the portfolio page. Contains a showProjectDetail method.
+  *
+  * Copyright 2019 Google LLC
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *  https://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  *
+  * @author: shradha-khapra
+  */
 
 /**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  * Shows details of a specified project.
+  * @param {!Event} event The event object received.
+  * @return {void}
+  */
+function showProjectDetail(event) {
+  const descriptions =
+      [
+       "A Machine Learning Bot classifier used for segregation of Bots and Humans on Twitter during Indian socio-political uproar.",
+       "A Machine Learning image classifier to compare accuracy of KNN (with SVM) model and Neural Networks model for character recognition.", 
+       "A portal for organizing information and making appointments with healthcare professionals.", 
+       "A Google Maps & Weather API based application for real-time weather info on Maps with additional functionalities."
```

Comment:
A comma is required at the end of this line. Ideally, we put a comma at the end of the last element of array in JS. This is required only if the closing bracket is not on the same line as the last element. This is because, while adding a new element at the bottom the editor need not worry about the presence of comma on the line above. In compiled languages, absence of comma throws an error during compilation. But in interpreted languages no error is thrown and the result can be disastrous.